src/Entity/ReactionRuleStorage.php

```diff
@@ -61,6 +69,19 @@ public function __construct(EntityTypeInterface $entity_type, ConfigFactoryInter
   }
 
   /**
+   * Gets the event manager.
+   *
+   * @return \Drupal\rules\Core\RulesEventManager
+   *   The event manager.
+   */
+  protected function eventManager() {
+    if (!$this->eventManager) {
+      $this->eventManager = new RulesEventManager($this->moduleHandler());
```

Comment:
there is no \Drupal::eventManager(); and not $container->get('event_manager').
Is event_manager a core thing?
I cannot find an example  where it is initialized